pkgdb2/lib/__init__.py

```diff
@@ -298,7 +302,7 @@ def get_acl_package(session, pkg_name, pkg_clt=None, eol=False):
         found in the database with the name ``pkg_name``.
 
     """
-    package = model.Package.by_name(session, pkg_name)
+    package = model.Package.by_name(session, pkg_name, namespace)
```

Comment:
In some functions, the arguments are ``(session, namespace, pkg_name, ....)`` and here it is ``(session, pkg_name, namespace)``.  It would help for long term maintainability and debugging if it was consistent.  I like the idea of having ``namespace`` come before ``pkg_name``.  Can we standardize on that?